FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -210,40 +210,99 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                            </msItem>
+                           <msItem xml:id="p1_i2.5.2">
+                               <locus target="116r"></locus>
+                               <title ref="LIT6355MEMiracle"></title>
+                           </msItem>
+                            <msItem xml:id="p1_i2.6">
+                                <locus from="116rb" to="117rb"></locus>
+                                <title ref="LIT6356MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.7">
+                                <locus from="117rb" to="118ra"></locus>
+                                <title></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.8">
+                                <locus from="118ra" to="119rb"></locus>
+                                <title ref="LIT6358MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.9">
+                                <locus from="119rb" to="119vb"></locus>
+                                <title ref="LIT6359MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.10">
+                                <locus from="119vb" to="120ra"></locus>
+                                <title ref="LIT6360MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.11">
+                                <locus from="120ra" to="120rb"></locus>
+                                <title ref="LIT6362MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.12">
+                                <locus from="120rb" to="121va"></locus>
+                                <title ref="LIT6363MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.13">
+                                <locus from="121va" to="121vb"></locus>
+                                <locus from="123ra" to="123vb"></locus>
+                                <title ref="LIT6365MESick"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.14">
+                                <locus from="124ra" to="127ra"></locus>
+                                <title></title>
+                                <note>No. 14 corresponds to No. 17 in <bibl><ptr target="bm:Turaiev1905Eustathii"/><citedRange unit="page">167-177</citedRange></bibl> and in
+                                    <bibl><ptr target="bm:Turaev1906Ewostatewos"/><citedRange unit="page">92-97</citedRange></bibl>.</note>
+                            </msItem>
+                            <msItem xml:id="p1_i2.15">
+                                <locus from="127ra" to="127va"></locus>
+                                <title></title>
```

Comment:
same as above. the information in the note is relevant for the work, not for the manuscript